tests/gocase/util/server.go

```diff
@@ -21,29 +21,39 @@ package util
 
 import (
 	"context"
+	"crypto/tls"
+	"crypto/x509"
 	"flag"
 	"fmt"
 	"net"
 	"os"
 	"os/exec"
 	"path/filepath"
+	"reflect"
 	"regexp"
 	"syscall"
 	"testing"
 	"time"
 
 	"github.com/go-redis/redis/v9"
+	"github.com/shirou/gopsutil/v3/process"
 	"github.com/stretchr/testify/require"
 )
 
 var binPath = flag.String("binPath", "", "directory including kvrocks build files")
 var workspace = flag.String("workspace", "", "directory of cases workspace")
 var deleteOnExit = flag.Bool("deleteOnExit", false, "whether to delete workspace on exit")
+var TLSServerName = flag.String("TLSServerName", "localhost", "server name for TLS connection")
+
+var EnableTLSTests = flag.Bool("enableTLSTests", false, "enable TLS-related test cases")
+var CliPath = flag.String("cliPath", "redis-cli", "path to redis-cli")
```

Comment:
If these options are in other places like `tls_test.go`, then `go test -enableTLSTests ./...` will fail for all tests except `tls_test.go`.